tests/test_gitlab.py

```diff
@@ -1323,6 +1341,17 @@ def test_sanitize_for_archive(self):
         self.assertEqual(headers, s_headers)
         self.assertEqual(payload, s_payload)
 
+        url = "http://example.com"
```

Comment:
This method could be splitted in two: `test_sanitize_for_archive_private_token` and `test_sanitize_for_archive_oauth_token`. The former is the old `test_sanitize_for_archive` while the new one contains your code.

What do you think ?